sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfig.java

```diff
@@ -59,6 +64,18 @@ public Double getTargetThroughputThreshold() {
         return this.targetThroughputThreshold;
     }
 
+    /**
+     * Get the throughput control group priority level.
+     * Priority level is used to determine which group will be throttled first when the total throughput of all groups exceeds the max throughput.
+     *
+     * Default PriorityLevel for each request is treated as High. It can be explicitly set to Low for some requests.
+     *
+     * Refer to https://aka.ms/CosmosDB/PriorityBasedExecution for more details.
```

Comment:
Do we have a public doc for the priority throttling? because it will just redirect me to https://learn.microsoft.com/en-us/azure/cosmos-db/set-throughput

Reply:
Yes. This page will be updated. It's currently in progress. @richagaur is working on it

Reply:
1. In the backend, Priority based throttling supports all kinds of operations. We will start by enabling on customer requests and enable for other operations later.
2. The result will show the same status code (429/3200) if the request fails due to priority based throttling.
3. Priority Based throttling adjusts the budget for low and high priority requests dynamically so that high priority requests can use the maximum possible amount of RUs. However, if the user wants to put limits on each throughput control group, they are free to do so.